### ISSUE TITLE
Artifacts with Local Contrast + Resize

### ISSUE THREAD

**Issue by Queatz** [Beep6581/RawTherapee#6184]
**Short description**
Artifacts appear with Local Contrast + Resize

**Steps to reproduce**
1. Turn on Local Contrast
2. Use the Resize to make the photo smaller
3. Artifacts appear (in saves + flickering in viewport)

Note: I believe using Resize just exaggerates the artifacts.

**Expected behavior**
No artifacts

**Additional information**
- Version of RawTherapee = dev branch local build + the last Nightly AppImage 
- Ubuntu 21.04

Exported from RawTherepee:
![DSCF1703-1](https://user-images.githubusercontent.com/540620/112408594-16557c80-8ce6-11eb-86df-683c5a37e966.jpg)

**Comment by chaimav** [comment]
I am unable to duplicate in the Widows build. Can you share an image and a pp3 where it happens?

**Comment by Queatz** [comment]
Yes here is that image: [DSCF1703.JPG.zip](https://github.com/Beep6581/RawTherapee/files/6206489/DSCF1703.JPG.zip)

**Comment by Thanatomanic** [comment]
@Queatz I also cannot reproduce, not with your pp3, nor by applying it to either a raw file, or a jpeg. 

Can you please say a bit more about your machine? And please post the contents of the `AboutThisBuild` file in the build directory.

**Comment by Queatz** [comment]
I don't seem to have that file? I only have AboutThisBuild.txt.in.  I built with `build-rawtherapee`.

I did see this in the build log, however:
```
[  0%] Creating AboutThisBuild.txt and other version-dependent files
-- git command found: /usr/bin/git
-- Git checkout information:
--     Commit description:    5.8-2829-g871d35c22
--     Branch:                dev
--     Commit:                871d35c22
--     Commit date:           2021-03-24
--     Commits since tag:     2748
--     Commits since branch:  2748
--     Version (unreliable):  5.8.2748
-- Build information:
--     Build OS:              Linux 5.11.0-11-generic x86_64
--     Build date:            Thu, 25 Mar 2021 01:47:07 +0000 UTC
--     Epoch:                 1616636827
--     UUID:                  2b4449ec-f9ed-46d4-a3bb-1ab79fa61cb6
-- CACHE_NAME_SUFFIX is "5-dev"
[  0%] Built target UpdateInfo
```

I am using Ubuntu 21.04.

Another thing, I don't remember seeing these artifacts using this same AppImage before, it may be possible that updating nvidia drivers or other components aren't playing nicely?

**Comment by Queatz** [comment]
Here is the original image if it helps: https://drive.google.com/file/d/1Boghhw_Nc7YHtBHA5pyF46aPX7plAOsz/view?usp=sharing

**Comment by Thanatomanic** [comment]
@Queatz 
> Can you please say a bit more about your machine?

What I mean is not only your OS, but what kind of CPU you have and amount of RAM. I still cannot reproduce, but since the file comes from a GFX 100 and is 50 megapixels there may be a memory issue.

**Comment by Queatz** [comment]
@Thanatomanic I have 32GB RAM with AMD Ryzen 9 5950x 16-core CPU and NVIDIA GeForce RTX 3080.  I definitely have RAM available.

I am also seeing these artifacts in the viewport: https://youtu.be/Uc6lKGnBWaw

**Comment by Floessie** [comment]
Tested with the JPG and PP3 provided on Debian Testing AMD64, Ryzen 5 3600, with current `dev` as well as with 871d35c22. Cannot reproduce. :frowning_face: 

We are still missing the `AboutThisBuild.txt`. It must be there in the build directory:

```
[  0%] Creating AboutThisBuild.txt and other version-dependent files
```

What about `find -name 'AboutThisBuild.txt'`?

HTH,
Flössie

**Comment by Queatz** [comment]
@Floessie 

Finally I found the AboutThisBuild.txt file:

```
Version: 5.8-2854-g1c9d1f522
Branch: dev
Commit: 1c9d1f522
Commit date: 2021-05-07
Compiler: cc 10.2.1
Processor: x86_64
System: Linux
Bit depth: 64 bits
Gtkmm: V3.24.2
Lensfun: V0.3.2.0
Build type: release
Build flags:  -std=c++11 -march=native -Werror=unused-label -Werror=delete-incomplete -fno-math-errno -Wno-attributes -Wall -Wuninitialized -Wcast-qual -Wno-deprecated-declarations -Wno-unused-result -Wunused-macros -fopenmp -Werror=unknown-pragmas -O3 -DNDEBUG -ftree-vectorize
Link flags:  -march=native
OpenMP support: ON
MMAP support: ON
Build OS: Linux 5.11.0-13-generic x86_64
Build date: Sun, 09 May 2021 19:51:37 +0000 UTC
Build epoch: 1620589897
Build UUID: aed5e965-c0f2-4241-a48b-622738dca904
```

**Comment by Queatz** [comment]
I built just using `cmake` w/o `./build-rawtherepee` and I no longer have the issue.

Edit: Scratch that.  It appears that the issue goes away with `Debug`, but is there with `release`

**Comment by Lawrence37** [comment]
I tried a dev release build in an Ubuntu 21.04 environment and could not reproduce the issue. The fact that only your release build shows the problem is a clue... I'm assuming your debug and release builds are based on the same commit?

**Comment by Queatz** [comment]
Yup exact same commit.  I actually may have previously upgraded various system components to 20.10.  I've been fully on Ubuntu 20.10 since it launched.

**Comment by Queatz** [comment]
One other thing, I did notice small artefacts on one photo I saved with the debug build (out of about 40.)  So it may be that the debug build is slower which causes the artefacts to not form as much.

**Comment by Lawrence37** [comment]
Tried with 20.10 and still couldn't reproduce :open_mouth:
